content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -477,6 +477,65 @@ runtime  recursively changes the SELinux label for all inodes (files and directo
 in the volume.
 The more files and directories in the volume, the longer that relabelling takes.
 
+{{< note >}}
+<!-- remove after Kubernetes v1.30 is released -->
```

Comment:
Is this right?

Reply:
ah this was pulled in from a merge conflict...